src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SortableColumnHeader.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class SortableColumnHeader
```

Comment:
Also, to avoid confusion, rename it to a helper or utility, as this doesn't represent any object.